sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/LogBase.java

```diff
@@ -35,15 +36,19 @@
 
     public static final String LOG_CHARSET = "utf-8";
 
+    public static final String LOG_TYPE_FILE = "file";
+    public static final String LOG_TYPE_CONSOLE = "console";
+
     private static final String DIR_NAME = "logs" + File.separator + "csp";
     private static final String USER_HOME = "user.home";
 
+    public static final String LOG_TYPE = "csp.sentinel.log.type";
```

Comment:
Maybe names like `csp.sentinel.record.log.output.type` or another brief one would be better? This could indicate this property is only for biz log (`RecordLog` and `CommandCenterLog`) and designed for the output type.

Reply:
Indicate the detail meaning is better, how about a little brief name `csp.sentinel.log.biz.output`, which first means log, then biz means(RecordLog and CommandCenterLog), then output means the output type, output to file or console.

In the future, we can support metric log config like `csp.sentinel.log.metric.output`.

As a choosing disorder, how about replace biz to system?
`csp.sentinel.log.system.output`
`csp.sentinel.log.biz.output`
`csp.sentinel.log.sys.output`
`csp.sentinel.log.system.output.type`
`csp.sentinel.log.biz.output.type`
`csp.sentinel.record.log.output.type`
`csp.sentinel.log.system.appender`

I prefer 1 or 2.
What's your opinion? @sczyh30 @jasonjoo2010

Reply:
Actually i prefer not to limit to record logging.

We can adjust a common interface in future for all sentinel modules i think. What's you plan @sczyh30 ?

Reply:
Actually i prefer not to limit to record logging.

We can adjust a common interface in future for all sentinel modules i think. What's you plan @sczyh30 ?

Reply:
> Actually i prefer not to limit to record logging.
> 
> We can adjust a common interface in future for all sentinel modules i think. What's you plan @sczyh30 ?

How about we separate this config parameter at present，since for metric log, as it may grows fast, it's Inappropriate output to console.

Reply:
> > Actually i prefer not to limit to record logging.
> > We can adjust a common interface in future for all sentinel modules i think. What's you plan @sczyh30 ?
> 
> How about we separate this config parameter at present，since for metric log, as it may grows fast, it's Inappropriate output to console.

I think `metric log` is a kind of structural storage but not a kind of logging. So just exclude it.

Reply:
This could be a temporary solution. I think after the logging interface is ready, it would be better to choose different output types by introducing the specific SPI implementation module, rather than putting it in the command line args.

Reply:
Agree. This is to provide a easy temporary way for user who want to ouput biz log to console : )